Classes/Utility/Dimension.php

```diff
@@ -49,7 +50,7 @@ public static function extractWidth($str)
      * @param string $str
      * @return string
      */
-    public static function extractHeight($str)
+    public static function extractHeight($str): \string
```

Comment:
"Fatal error: Scalar type declaration 'string' must be unqualified"
-> should be ": string" instead of ": \string"